### PR TITLE
Remove some old files, cleanup docs.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

**Comment by dlorenc** [comment]
cc @aaron-prindle

File: CONTRIB.md [hunk 1]
```diff
@@ -1,12 +1,31 @@
-# How to build this base image
+# How to build these base images
 
-`make all` will generate the builder image and create a fresh debootstrap rootfs
-for the debian version specified by `DEBIAN_SUITE` (defaults to jessie). The
-bzipped tarball which results is then used to create a bare debian docker image.
+We use `bazel` to build most of the images in this repository, so that we can build them reproducibly.
+To learn about how we generate reproducible images, see [the design doc](./reproducible/README.md).
 
+To build all images, use:
```
Comment (aaron-prindle) [inline CONTRIB.md]: it might be useful to list the docker images that this will produce:
bazel/reproducible:debian{#}
bazel/reproducible:ubuntu

**Comment by aaron-prindle** [comment]
LGTM

File: CONTRIB.md (hunk 1 above)
Comment (dlorenc) [reply CONTRIB.md]: I thought about that, but then spent awhile trying to figure out how to keep it up to date automatically, then gave up :(

**Pull request merged**